apps/dh/api-dh/source/DataHub.WebApi/GraphQL/Resolvers/MarketParticipantResolvers.cs

```diff
@@ -50,7 +50,7 @@ public async Task<IEnumerable<GridAreaDto>> GetGridAreasAsync(
                 .Distinct()
                 .Select(async gridAreaId => await dataLoader.LoadAsync(gridAreaId)));
 
-        return gridAreas.OrderBy(g => g.Code);
+        return gridAreas.Select(gridArea => gridArea!).OrderBy(g => g.Code);
```

Comment:
Why do we need the null-forgiving operator here? Those always seem like code smell to me.

Reply:
Fixed.